tests/test_media.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/* Media Endpoint and Upload Tests inspired by REST API Attachment Endpoint */
+
+class Micropub_Media_Test extends WP_UnitTestCase {
```

Comment:
awesome!

maybe also consider testing a request that's missing a file.

Reply:
Also will do.